ddtrace/tracer/span_test.go

```diff
@@ -344,6 +344,26 @@ func TestSpanErrorNil(t *testing.T) {
 	assert.Equal(nMeta, len(span.Meta))
 }
 
+func TestUniqueTagKeys(t *testing.T) {
+	assert := assert.New(t)
+	span := newBasicSpan("web.request")
+
+	//check to see if setMeta correctly wipes out a metric tag
+	span.SetTag("foo.bar", 12)
+	span.SetTag("foo.bar", "val")
+
+	assert.Equal(0.0, span.Metrics["foo.bar"])
```

Comment:
Rather than checking for the zero values, we should use [`assert.NotContains`](https://godoc.org/github.com/stretchr/testify/assert#NotContains) to make sure the value is not present in the map. An empty string or a zero are valid values that could be in the map, but would be wrong.